### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone I: create animals table

In this PR I complete following steps:

 - Create a database named `vet_clinic`.
 - Create a table named `animals` with the following columns [(schema.sql)](https://github.com/anthonymr/vet-clinic/blob/create-animals-table/schema.sql)
   - `id`: integer
   - `name`: string
   - `date_of_birth`: date
   - `escape_attempts`: integer
   - `neutered`: boolean
   - `weight_kg`: decimal
 - Insert test data into the `animals` table [(data.sql)](https://github.com/anthonymr/vet-clinic/blob/create-animals-table/data.sql).
 - Write queries for the following quests [(queries.sql)](https://github.com/anthonymr/vet-clinic/blob/create-animals-table/queries.sql):
   - Find all animals whose name ends in "mon".
   Query_1
   - List the name of all animals born between 2016 and 2019.
   Query_2
   - List the name of all animals that are neutered and have less than 3 escape attempts.
   Query_3
   - List the date of birth of all animals named either "Agumon" or "Pikachu".
   Query_4
   - List name and escape attempts of animals that weigh more than 10.5kg
   Query_5
   - Find all animals that are neutered.
   Query_6
   - Find all animals not named Gabumon.
   Query_7
   - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
   Query_8
 
ℹ️ To run the project:

1. Clone the repo by running `git clone git@github.com:anthonymr/vet-clinic.git`.
2. Go to the project folder with `cd vet-clinic`.
3. Run the SQL scripts on your local PostgreSQL instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,234 +26,125 @@ After you're finished please remove all the comments and instructions!
...
-  - [Answer_2]
+I would like to thank to [Microverse](https://www.microverse.org/) for provide the base idea for this project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Lizdev-05) [inline README.md]: Kindly check the MIT link. The link is broken presently 😞

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @anthonymr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter error 👍 
- Great implementation 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (anthonymr) [reply README.md]: Hi @Lizdev-05 thanks very much for you review.

**Review by gemmen29 — APPROVED** [review]
Hi @anthonymr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**